### PR TITLE
Android - Event detail page

### PR DESCRIPTION
**Pull request by ybedirhanpak**
- Implement event detail page
- Connect event detail page with search fragment

### CODE REVIEW

File: android/app/src/main/java/com/bounswe2020group3/paperlayer/event/EventDetailFragment.kt [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                EventDetailFragment().apply {
+                    arguments = Bundle().apply {
+                        putInt(ARG_EVENT_ID, eventID)
+                    }
+                }
+    }
+
+    override fun showLoading() {
+        progressBarEvent.visibility = View.VISIBLE
+        layoutEvetWrapper.visibility = View.GONE
```
Comment (yusufbayam) [inline android/app/src/main/java/com/bounswe2020group3/paperlayer/event/EventDetailFragment.kt:71]: I think there is a typo here, maybe layoutEventWrapper?

File: android/app/src/main/java/com/bounswe2020group3/paperlayer/event/EventDetailFragment.kt [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    }
+
+    override fun hideLoading() {
+        progressBarEvent.visibility = View.GONE
+        layoutEvetWrapper.visibility = View.VISIBLE
```
Comment (yusufbayam) [inline android/app/src/main/java/com/bounswe2020group3/paperlayer/event/EventDetailFragment.kt:76]: I think there is a typo here, maybe layoutEventWrapper?

File: android/app/src/main/res/layout/fragment_event.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:visibility="gone"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <LinearLayout
+        android:id="@+id/layoutEvetWrapper"
```
Comment (yusufbayam) [inline android/app/src/main/res/layout/fragment_event.xml:21]: I think there is a typo here, maybe layoutEventWrapper?

**Review by yusufbayam — COMMENTED** [review]
Looking good, can be merged after fixing typos.

**Review by rambo250 — COMMENTED** [review]
Good Work

File: android/app/src/main/java/com/bounswe2020group3/paperlayer/event/EventDetailFragment.kt (hunk 1 above)
Comment (ybedirhanpak) [reply android/app/src/main/java/com/bounswe2020group3/paperlayer/event/EventDetailFragment.kt:71]: Yes, definitely. I'll fix them. Thank you :)

**Review by yusufbayam — APPROVED** [review]
Nice work, thanks.

**Pull request merged**